pages/price-feeds/error-codes/evm.mdx

```diff
@@ -0,0 +1,21 @@
+# EVM Error Codes
+
+[PythErrors.sol](https://github.com/pyth-network/pyth-crosschain/blob/a888ba318c0325c29070eaf5afcc3a4d443b058c/target_chains/ethereum/sdk/solidity/PythErrors.sol#L8)
+contains the errors that are used in the Pyth Network's EVM contracts.
```

Comment:
I suggest wording this like "The following table contains ... . This information is derived from PythErrors.sol in the Pyth SDK and can be used to decode error codes programmatically"